pyme-depends-strict/conda_build_config.yaml

```
@@ -0,0 +1,8 @@
+python:
+  # - 2.7.*
+  - 3.6.*
+  - 3.7.*
```

```suggestion
  # - 3.7.*
```

These are derived for py3.6 right - I think there is very little chance of them working for py37 as well